common/test/keyboards/baseline/k_020___deadkeys_and_backspace.kmn

```diff
@@ -22,3 +23,6 @@ group(main) using keys
 + '4' > dk(4a) dk(4b)
 + '5' > 'a' dk(5a) dk(5b) 'b'
 + '6' > 'a' dk(6a) dk(6b) 'b' dk(6c) dk(6d)
++ '7' > 'c' dk(7) 'd'
+'c' 'd' + [K_BKSP]       > ' fail'
+'c' dk(7) 'd' + [K_BKSP] > ' ok'
```

Comment:
This is a slightly different test case -- testing the matching of rules rather than the default backspace behaviour. Can we move it to a separate keyboard and document accordingly?

Reply:
If the actual test uses the context `c dk(u) d f` and then processes two backspaces, I think that would be adequate for the test.  It's just that there's no special rule for the `f`... and the first backspace _would_ be default.  Keeping in mind that the actual error bug resulted from the _lack_ of a deadkey after a backspace triggering a _backspace_-based rule, I'm personally fine with the form of the check.

Thing is, this test keyboard _is_ called "deadkeys and backspace".  You sure it doesn't fit here?  The other tests handle deadkeys between the latest context char and the context char before it; this test handles a deadkey based one char earlier than _that_.

Reply:
yes, having two backspaces is definitely better since the deadkey removal could happen before or after processing the first one. I updated the test.